### ISSUE TITLE
Add default groups collapse/expand option

### ISSUE THREAD

**Issue by SinaniG1996** [fabiospampinato/vscode-projects-plus#57]
Hello. Now groups in sidebar view are expanded by default and there is no option in settings to change this behaviour. 

Could you please add default collapse/expand groups option in sidebar view to extension settings?

**Comment by colorful-tones** [comment]
Thanks for all the hard work on this extension! 👏 

I would love to have this feature as well.

**Comment by azchatlanin** [comment]
Yes, you need to add state saving. When there are many projects and all groups are deployed it is very inconvenient! Thanks for you work!

**Comment by robkus** [comment]
Great extension, thanks! I would love this feature very much, too. Either by state saving or just setting to collapse the structure always when displayed. It is faster to expand the desired group from a clearly arranged state than to scroll or collapse the groups first :)

**Comment by inu-web** [comment]
HI, 
I would highly appreciate this feature too, because of many projects in the treeview
e.g. a parameter **"collapsed" : true/false**    would be very helpful  ...
thanks a lot for this gread extension!
regards Gerald

**Example Config:**
{
  "groups": [
   {
      "name": "GROUP-1 (open by default)",
      **"collapsed": false,**
      "projects": [
        {
          "name": "PROJECT1",
          "description": "",
          "path": "//MYPATH/Folder1/Folder1-1"
        },
        {
          "name": "PROJECT2",
          "description": "",
          "path": "//MYPATH/Folder2/Folder2-1"
        }
      ]
    },
   {
      "name": "GROUP-2  (collapsed by default)",
      **"collapsed": true,**
      "projects": [
        {
          "name": "PROJECT-3",
          "description": "",
          "path": "//MYPATH/Folder1/Folder1-1"
        },
        {
          "name": "PROJECT-4",
          "description": "",
          "path": "//MYPATH/Folder2/Folder2-1"
        }
      ]
    }
}

**this would then result in the following tree in Projects+**

▼ GROUP1  (open by default)
          PROJECT-1
          PROJECT-2
► GROUP-2 (collapsed by default)
